src/patternfly/components/DualListSelector/dual-list-selector-item.hbs

```diff
@@ -1,4 +1,4 @@
-<{{#if dual-list-selector-item--type}}{{dual-list-selector-item--type}}{{else}}button{{/if}} class="pf-c-dual-list-selector__item{{#if dual-list-selector-item--HasCheck}} pf-m-check{{/if}}{{#if dual-list-selector-item--IsSelected}} pf-m-selected{{/if}}{{#if dual-list-selector-item--modifier}} {{dual-list-selector-item--modifier}}{{/if}}"
+<{{#if dual-list-selector-item--type}}{{dual-list-selector-item--type}}{{else}}button{{/if}} class="pf-c-dual-list-selector__item{{#if dual-list-selector-item--HasCheck}} pf-m-check{{/if}}{{#if dual-list-selector-item--IsSelected}} pf-m-selected{{/if}}{{#if dual-list-selector-item--IsDisabled}} pf-m-disabled{{/if}}{{#if dual-list-selector-item--modifier}} {{dual-list-selector-item--modifier}}{{/if}}"
```

Comment:
When the item is a `<button>`, you should only need to add a `disabled` attribute, then all you need to do is style the text via `:disabled`.

But this item can also be a `<div>` for both expandable and checkbox items. Maybe in those cases, it's appropriate to add `.pf-m-disabled` here, remove the `tabindex` from this element, add `disabled` to the check,  and style the toggle and text via `.pf-m-disabled` as you've done here.